build.gradle

```diff
@@ -29,6 +29,20 @@ repositories {
     maven { url "https://mvnrepository.com/artifact/org.zeromq/jeromq" }
 
     maven { url uri ("../SeriouslyCommonLib/lib/")}
+
+    maven {
+        url = uri("https://maven.pkg.github.com/Mechanical-Advantage/AdvantageKit")
+        credentials {
```

Comment:
⭐  can you add a comment about where these came from in case we ever have to update them?